clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-metal_machine_set.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build01-9hdwj
+  name: build01-9hdwj-worker-us-east-1a-metal
```

Comment:
To generate this I took the existing `search` machineset and just did `s/search/metal/` btw.  Per the GH comment I also commented out the taint.